peers/external_handler.go

```diff
@@ -101,7 +101,7 @@ func (h *RelayerExternalHandler) HandleInbound(_ context.Context, inboundMessage
 }
 
 func (h *RelayerExternalHandler) Connected(nodeID ids.NodeID, version *version.Application, subnetID ids.ID) {
-	h.log.Debug(
+	h.log.Info(
```

Comment:
(not a blocker) are we keeping this at `Info` level or was this just for testing? I'd prefer having both `Connected` and `Disconnected` events at the same log level in general. I guess we can match them on the metrics side but if diagnosing anything network related it's nice to also know the interval .

Reply:
This was a test artifact. Removed now.